website/source/docs/guides/acl-migrate-tokens.html.md

```diff
@@ -0,0 +1,453 @@
+---
+layout: "docs"
+page_title: "ACL Token Migration"
+sidebar_current: "docs-guides-acl-migrate-tokens"
+description: |-
+  Consul 1.4.0 introduces a new ACL system with improvements for the security and
+  management of ACL tokens and policies. This guide documents how to upgrade
+  existing (now called "legacy") tokens after upgrading to 1.4.0.
+---
+
+# ACL Token Migration
+
+Consul 1.4.0 introduces a new ACL system with improvements for the security and
+management of ACL tokens and policies. This guide documents how to upgrade
+existing (now called "legacy") tokens after upgrading to 1.4.0.
+
+Since the policy syntax changed to be more precise and flexible to manage, it's
+necessary to manually translate old tokens into new ones to take advantage of
+the new ACL system features. Tooling is provided to help automate this and this
+guide describes the overall process.
+
+~> **Note:** **1.4.0 retains full support for "legacy" ACL tokens** so upgrades
+from Consul 1.3.0 are safe. Existing tokens will continue to work in the same
+way for at least two "major" releases (1.5.x, 1.6.x, etc; note HashiCorp does
+not use SemVer for our products).
+
+This document will briefly describe [what changed](#what-changed), and then walk
+through the [high-level migration process options](#migration-process), finally
+giving some [specific examples](#migration-examples) of migration strategies.
+
+## New ACL System Differences
+
+The [ACL guide](/docs/guides/acl.html) and [legacy ACL
+guide](/docs/guides/acl-legacy.html) describes the new and old systems in
+detail. Below is a summary of the changes that need to be considered when
+migrating legacy tokens to the new system.
+
+### Token and Policy Separation
+
+You can use a single policy in the new system for all tokens that share access
+rules. For example, all tokens created using the clone endpoint in the legacy
+system can be represented with a single policy and a set of tokens that map to
+that policy.
+
+### Rule Syntax Changes
+
+The most significant change is that rules with selectors _no longer prefix match
+by default_. In the legacy systems the following rules would grant access to
```

Comment:
`In the legacy systems` or `In the legacy system` pretty sure it should be system (singular) but thought I'd check